AMTGBot/Bot.cs

```diff
@@ -11,12 +14,16 @@ public class Bot
         public ITelegramBotClient BotClient { get; }
         public BotConfig BotConfig { get; }
         public ILatexRenderer LatexRenderer { get; }
+        public Logger Logger { get; private set; }
 
         public Bot(ITelegramBotClient botClient, BotConfig botConfig, ILatexRenderer latexRenderer)
         {
             BotClient = botClient;
             BotConfig = botConfig;
             LatexRenderer = latexRenderer;
+
+            Logger = LogManager.GetCurrentClassLogger();
```

Comment:
Again, if following SOLID, one should pass this as an argument when creating an instance of Bot

Reply:
Agree.